bucket/deew.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "3.0.1",
+    "description": "Dolby Encoding Engine Wrapper",
+    "homepage": "https://github.com/pcroland/deew",
+    "license": "MIT License",
+    "suggest": {
+        "ffmpeg": [
+            "ffmpeg",
+            "versions/ffmpeg-nightly"
+        ]
+    },
+    "url": "https://github.com/pcroland/deew/releases/download/3.0.1/deew.exe",
+    "hash": "c621b9af9b037f78be4626c5b592dbf04311d214bc7ee46c0808eddd4f9d5ca3",
+    "bin": "deew.exe",

```

Comment:
`deew.exe` is a 64bit program, use `architecture`.

Reply:
added